.changeset/orange-seas-knock.md

```diff
@@ -0,0 +1,6 @@
+---
+'@graphql-mesh/config': patch
+'@graphql-mesh/types': patch
+---
+
+Fix incorrect import code generated based on the `logger` configuration option and the documentation suggesting to pass a logger instance instead of a file path.
```

Comment:
Still doesn't say that the only valid option is a string. You really don't have to overthink it, as simple as the following works:

```suggestion
`logger` configuration option only accepts a string
```